jadx-cli/src/main/java/jadx/cli/JadxCLIArgs.java

```diff
@@ -504,7 +504,7 @@ public static class UseKotlinMethodsForVarNamesConverter implements IStringConve
 		@Override
 		public UseKotlinMethodsForVarNames convert(String value) {
 			try {
-				return UseKotlinMethodsForVarNames.valueOf(value.replace('-', '_').toUpperCase());
```

Comment:
Note that this one already did the `- -> _` replacement. 

One thing I'm unclear on is if the `toUpperCase()` should include the `Locale.ROOT` constant that the `enumValuesString` passes.. though I don't follow what that's doing - the characters in the enums are all ASCII to begin with?